ouroboros-consensus/src/Ouroboros/Consensus/Mempool/TxSeq.hs

```diff
@@ -160,5 +164,5 @@ fromTxSeq (TxSeq ftree) = fmap
 -- transaction's ticket number.
 appendTx :: TxSeq tx -> tx -> TxSeq tx
 appendTx ts tx = case viewBack ts of
-  Nothing                           -> Empty :> TxTicket tx (TicketNo 0)
+  Nothing                           -> Empty :> TxTicket tx zeroTicketNo
```

Comment:
I think this will need to start at 1 not 0, because the operation we provide is to find all txs with ticket numbers _strictly greater_ than the given ticket (which is the right operation to provide since we want to see all new txs after the last one we've seen so far). So we would not find the first tx if we give it ticket 0.

Reply:
Oops. Thanks.